rust/connlib/snownet/src/node.rs

```diff
@@ -832,19 +832,34 @@ where
         packet: &'p [u8],
         now: Instant,
     ) -> ControlFlow<(), (SocketAddr, &'p [u8], Option<Socket>)> {
+        if from.port() != 3478 {
+            // Relays always send & receive from port 3478.
+            return ControlFlow::Continue((from, packet, None));
+        }
```

Comment:
I had to add this so we don't discard STUN messages from other peers as ones from "not connected relays". I think this is fine, relays are expected to always listen on port 3478.

Reply:
There is of course the edge case that legitimate STUN traffic from a client will come from this port.

Is it doable / easy to check against the existing Relay IPs here as well?

Reply:
Well, we do see GCP using consistent low-number ports per node (e.g. `1056`) so it's really up to the NAT. But I suppose the chance is low enough (0.0015%) that this probably isn't worth it.

Reply:
I'll add a comment.